tests/layer_tests/pytorch_tests/pytorch_layer_test_class.py

```diff
@@ -168,8 +168,9 @@ def convert_directly_via_frontend(self, model, example_input, trace_model, dynam
                 model = torch.jit.trace(model, example_input)
             else:
                 model = torch.jit.script(model)
-        print(model.inlined_graph)
         decoder = TorchScriptPythonDecoder(model, freeze=freeze_model)
+        model = decoder.pt_module
+        print(model.inlined_graph)
```

Comment:
why do you need print in the tests?

Reply:
This print existed before, If I understand properly, it is used to print result of TorchScript conversion in case of some failure debugging. One thing I didn't though about before is that previously TorchScript would show also if TorchScriptPythonDecoder would fail, maybe moving it was bad idea